content/docs/user-guide/appendix/cvars/_index.md

```diff
@@ -1,27 +1,56 @@
 ---
-title: CVARs
+title: Console variables (CVARs)
 ---
 
-Console variables (CVARs) can be used to set various editor and runtime options in O3DE. O3DE ships with many built-in CVARs. 
+Console variables (CVARs) can be used to set various editor and runtime options in O3DE. O3DE ships with many built-in CVARs but also supports options for configuring and grouping your own.
 
 ## Working with CVARs
-You can define your own CVARs in code using the [CVAR macro functions](/docs/user-guide/programming/az-console/#console-variables-cvars).
+
+You can define your own CVARs in code using the [CVAR macro functions](/docs/user-guide/programming/az-console/#console-variables-cvars). You can also group CVARs together in [console variable groups](/docs/user-guide/editor/console-cvars-commands/#console-variable-groups), which allow you to change multiple CVARs at once. 
 
 You can invoke CVARs in the Editor via the [console](/docs/user-guide/editor/console/).
 
 You can also pass CVARs on the command line when launching O3DE binaries via `--{cvar name} {value}` or `--{cvar name}={value}`. For example, `ServerLauncher.exe --sv_somebool=true`.
 
+Additionally, you can add CVARs to [configuration files](/docs/user-guide/editor/console/#configuring-console-variables-in-configuration-files).
+
+### Discovering CVARs
+
+The easier way to find the CVARs settable in the current project is via the [console variables window](/docs/user-guide/editor/console/#viewing-the-console-window). Bringing up the console window will list all the CVARs in the current project.
```

Comment:
When I read "the easier way" it sounded like it implied there was another (less easy?) method for finding cvars above. This is the only way to find cvar described, so maybe i'd say... 
"The way to find CVARs..." or 
"The easiest way to find CVARS..." or 
"Besides searching the code base for CVAR, the easier way to find CVARs..."

Reply:
You can discover cvars via the dump cvar commands and through the auto complete functionality in the console, so this is the easier way, but I can definitely rewrite.